runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/blog/BreadcrumbsSection/BreadcrumbsSection.tsx

```diff
@@ -1,9 +1,92 @@
-import React from 'react';
+import React, { FC } from 'react';
+
+import Image from 'next/image';
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+
+import robotIcon from '#public/images/icons/toy-orange.svg';
+import If from '#src-app/components/utils/If';
+import Typography from '#src-landing/components/Typography';
+import { capitalizeFirstLetter } from '#src-landing/utils/utils';
 
 import styles from './BreadcrumbsSection.module.scss';
+import { BreadcrumbsSectionProps } from './BreadcrumbsSection.types';
+
+const BreadcrumbsSection: FC<BreadcrumbsSectionProps> = ({
+    postTitle,
+}) => {
+    const router = useRouter();
+    const { pathname, asPath } = router;
+    const queryChars = ['?', '&', '='];
+    const uncoveredUrlPart = 'post';
+
+    const relevantPath = queryChars
+        .filter(
+            param => 
+                asPath.indexOf(param) !== -1
+        )
+        .length === 0 ? asPath : pathname;
+
+    const rawBreadcrumbs: string[] = relevantPath
+        .split('/')
+        .filter(
+            (breadcrumb) => breadcrumb !== ''
+        );
```

Comment:
I suggest to filter out `post` part in here rather than doing so in the map function

Reply:
It will remove one nesting from `breadcrumbs` variable return area

Reply:
not that simple, but done